src/applications/simple-forms/mock-simple-forms-patterns/arrayBuilder/components/ArrayBuilderCancelAddingButton.jsx

```diff
@@ -0,0 +1,117 @@
+/**
```

Comment:
these 'ArrayBuilder' specific components and helpers will eventually be moved to forms-system once we're happy with the pattern after we've built it all